src/main/java/org/cactoos/collection/HeadOf.java

```diff
@@ -27,7 +27,7 @@
 import org.cactoos.iterable.IterableOf;
 
 /**
- * Skipped collection.
+ * Head portion of collection.
```

Comment:
@Vatavuk Just renaming this one won't make it a `HeadOf`, I know @yegor256 suggested that in the original request, but in the next sentence you have:
> HeadOf will make an iterable of a certain head portion of the iterable

So what I was expecting is `HeadOf` that passes this test:
```
MatcherAssert.assertThat(
            new HeadOf<>(
                 2,
                 new IterableOf<>("one", "two", "three", "four", "five")
             ),
             Matchers.contains(
                 "one",
                 "two"
             )
);
```

So the opposite of `Skipped` - a collection that contains just the first `N` entries from the source collection.